docs/designs/output.md

```diff
@@ -79,44 +107,73 @@ Different files can share the same `{site-url}` or `{site-path}` due to versioni
     | `site-path` | api/system.string.json |
     | `output-path` | dotnet/01ddf122/api/system.string.json |
 
-- Content for static rendering using pretty url
+- Content for static hosting using pretty URL(OutputType: `Html`):
 
     | | |
     |------ |----|
-    | `url` | https://docs.microsoft.com/en-us/dotnet/api/system.string/ |
-    | `site-url` | /dotnet/api/system.string/ |
+    | `url` | https://self-hosting/en-us/dotnet/01ddf122/api/system.string/ |
```

Comment:
Add a note that for static page versioning, better use _moniker_ in the URL as it is human readable. That may require copying the same content to different output locations.

For the time been, we use moniker hash because the URL does not appear in PDF.
```suggestion
    | `url` | https://self-hosting/en-us/dotnet/v1/api/system.string/ |
```